resources/eventing/charts/controller/values.yaml

```
@@ -80,9 +80,20 @@ jetstream:
   streamName: sap
   # Subject prefix for the JetStream stream when eventTypePrefix is not provided.
   subjectPrefix: kyma
-  # When should events be deleted from the stream:
-  # interest: when all known observables have acknowledged a message it can be removed.
-  # limits: messages are retained until any given limit is reached. This can be configured via maxMessages and maxBytes.
+  # Retention policy determines when messages should be deleted from the stream:
+  # (more info https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive#stream-limits-retention-and-policy):
+  # - interest: when all known observables have acknowledged a message it can be removed.
+  # - limits: messages are retained until any given limit is reached. This can be configured via maxMessages and maxBytes.
```

```suggestion
  # - limits: Retain messages until any given limit is reached. Configure limits with maxMessages and maxBytes.
```
Active voice > passive voice

---

Done.